doc/tests/manual/reports/230725.md

```diff
@@ -0,0 +1,292 @@
+# Manual Test Report (pre v0.3 case study release)
+
+## General
+
+- Tester: Aydan Namdar Ghazani
+- Date/Time: 25 July 2023
+- Duration: 15 minutes
+- Commit/Tag: [af6b307](https://github.com/ElektraInitiative/PermaplanT/tree/af6b3070d285f273e6c173236a3c634eb4c827c3)
+- Planned tests: **13**
+- Executed tests: **13**
+- Passed tests: **10**
+- Failed tests: **3**
+
```

Comment:
We should also write here if the setup was local or if it was dev.permaplant.net.
And tests with problems should be here, too.

Reply:
I totally forgot about dev.permplant.net. Probably better to test on dev. since the database is one, instead of having different ones locally.

Reply:
I think these tests (and future ones too) should rarely be data dependent.